deploy/docker-compose.yml

```diff
@@ -47,6 +48,7 @@ services:
       CORS_ALLOWEDORIGINS: "${PUBLIC_ORIGIN:-http://localhost:${INTERNAL_HOST_PORT:-1235}}"
     volumes:
       - ${SHARED_ROOT:-/home}:${SHARED_ROOT:-/home}
+      - /var/folders:/var/folders
```

Comment:
This will not work on linux as there is no `/var/folders` folder. Please use variable in similar way as 
```
${SHARED_ROOT:-/home}:${SHARED_ROOT:-/home}
```

And write to README.md some documentation about it.